packages/i18n/README.md

```diff
@@ -46,51 +46,19 @@ You can manage translations in the `locales` directory.
 }
 ```
 
-## Delete or Add a new language
+`locales/pl/messages.json`
 
-When you want to delete or add a new language, you don't need to edit some util files like `lib/types.ts` or `lib/getMessageFromLocale.ts`. 
-That's because we provide a script to generate util files automatically by the `generate-i18n.mjs` file.
-
-Following the steps below to delete or add a new language.
-
-### Delete a language
-
-If you want to delete unused languages, you can delete the corresponding directory in the `locales` directory.
-
-```
-locales
-├── en
-│   └── messages.json
-└── ko // delete this directory
-    └── messages.json 
-```
-
-Then run the following command. (or just run `pnpm dev` or `pnpm build` on root)
-
-```bash
-pnpm genenrate-i8n
-```
-
-### Add a new language
-
-If you want to add a new language, you can create a new directory in the `locales` directory.
-
-```
-locales
-├── en
-│   └── messages.json
-├── ko
-│   └── messages.json
-└── ja // create this directory
-    └── messages.json // and create this file 
+```json
+{
+  "helloWorld": {
+    "message": "Witaj, Świecie!"
+  }
+}
```

Comment:
I think there are too many language examples 🤔 Is it necessary?

Reply:
I've done it because i think you want to have examples for all predefined languages